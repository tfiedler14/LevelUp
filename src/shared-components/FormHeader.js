import * as React from 'react';
import { Text } from 'react-native';
import EStyleSheet from 'react-native-extended-stylesheet';

export const FormHeader = ({ title }) => {
  return <Text style={styles.header}>{title}</Text>;
};

const styles = EStyleSheet.create({
  header: {
    paddingBottom: '1rem',
    fontWeight: 'bold',
    fontSize: '1rem',
    color: 'white'
  }
});
