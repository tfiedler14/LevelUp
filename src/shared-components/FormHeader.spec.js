import React from 'react';
import { shallow } from 'enzyme';
import { FormHeader } from './FormHeader';

describe('FormHeader', () => {
  it('should render FormHeader correctly', () => {
    const component = shallow(<FormHeader title={""} />);

    expect(component).toMatchSnapshot();
  });
});
