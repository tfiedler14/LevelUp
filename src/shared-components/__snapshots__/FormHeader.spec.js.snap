// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormHeader should render FormHeader correctly 1`] = `<Text />`;
